_components/validation/guidance/implementation.md

```diff
@@ -1,7 +1,7 @@
-- The validation component is intended primarily for usability, not as a robust security solution, since all the validation logic occurs on the client-side. The validation should be "mirrored" on the server-side for security purposes.
-- Input fields which have custom validation logic can automatically provide helpful feedback to users if they are assigned a `data-validation-element` attribute set to a CSS selector that uniquely identifies a `.usa-checklist`, e.g. `data-validation-element="#validate-code"`.
-- For each kind of validation you'd like to provide feedback on:
-  1. Come up with a name for the validator, e.g. `uppercase`. It shouldn't have any spaces in it.
-  2. On one of the list elements in the `.usa-checklist`, set the `data-validator` attribute to the name of the validator, e.g. `data-validator="uppercase"`. This is the list item that will appear "checked" when the validator's condition is met.
-  3. On the input field, add a field called data-validate-validator name and set its value to a [regular expression](https://regexone.com/) (described on RegexOne) that represents whether the validator's condition is met, e.g. `data-validate-uppercase="[A-Z]"`.
+- **Mirror this validation on the server side.** The validation component is intended primarily for usability, not as a robust security solution, since all the validation logic occurs on the client-side. The validation should be "mirrored" on the server-side for security purposes.
+- **Connect the input element to your validation checklist.** Set the value of your input's `data-validation-element` to the `id` of its related `.usa-checklist`. For example, if the `.usa-checklist` element has an `id` of `validate-code`, use `data-validation-element="validate-code"`.
+- **Customize the validation attributes.** For each type of validation you'd like to provide feedback on:
+  1. Come up with a custom name for your validator, e.g. `uppercase`. This name shouldn't have any spaces in it.
+  2. On the appropriate list item in `.usa-checklist`, set the `data-validator` attribute to the name of your custom validator, e.g. `data-validator="uppercase"`. This is the list item that will appear "checked" when the validator's condition is met.
+  3. On the input element, add an attribute called `data-validate-[validator name]`, where `[validator name]` is the custom name you created in step 1. Define this attribute with a [regular expression](https://regexone.com/) (described on RegexOne) that represents whether the validator's condition is met. For example, if your custom validator is called `uppercase`, use `data-validate-uppercase="[A-Z]"`.
```

Comment:
Is this all happening in one code block? If so, can we provide an example of a "finished" setup?

Reply:
It is all happening in one code block. The guidance here is giving an explanation of the code block that is already included inside the "Component Code" accordion above. Do you think it would be helpful to repeat it down here as well? Or maybe a reference to the component code section?

Reply:
I think a reference could be helpful!

Reply:
As in, "You can see a correctly configured validation in the xyz code block {linked or described}